src/vstest.console/Resources/Resources.resx

```diff
@@ -666,4 +666,18 @@
   <data name="UseVsixExtensionsDeprecation" xml:space="preserve">
     <value>/UseVsixExtensions is getting deprecated. Please use /TestAdapterPath instead.</value>
   </data>
+  <data name="ListFullyQualifiedTestsHelp" xml:space="preserve">
```

Comment:
Since `/listFullyQualifiedTests` is only available for compatibility, please skip the documentation.

Reply:
Agree. I did not add the HelpPriority for these and because they were not showing up in the help text I thought this will serve more of a guidance for our team. But yes we can remove this.